config/database.yml

```diff
@@ -22,10 +22,14 @@ default: &default
 development:
   <<: *default
   database: sulbib_development
+  host: 127.0.0.1
```

Comment:
can we put these in default?

Reply:
Yes. I had thought default was using unix sockets, but mistaken.